packages/remix-dev/compiler/js/write.ts

```diff
@@ -0,0 +1,17 @@
+import * as path from "node:path";
+import type * as esbuild from "esbuild";
```

Comment:
Any reason why we're not just importing `OutputFile` type here?

```suggestion
import type { OutputFile } from "esbuild";
```

Reply:
This was leftover after adapting the code from `packages/remix-dev/compiler/server/write.ts`.